h5p-bildetema/src/components/Bildetema/Bildetema.tsx

```diff
@@ -22,6 +22,13 @@ type BildetemaProps = {
 
 export const Bildetema: React.FC<BildetemaProps> = ({ defaultLanguages }) => {
   const { isLoading: isLoadingData, data } = useQuery(["dataFromDB"], getData);
+  const [showLoadingLabel, setShowLoadingLabel] = useState(false);
+
+  React.useEffect(() => {
+    setTimeout(() => {
+      setShowLoadingLabel(true);
```

Comment:
We should avoid re-rendering if the data has already been loaded.

```suggestion
      if (isLoadingData) {
        setShowLoadingLabel(true);
      }
```